api-service/service/main.tf

```diff
@@ -109,63 +109,50 @@ resource "aws_ecs_task_definition" "main" {
     },
     "environment": [
       {
-        "name": "RDS_DB_NAME",
-        "value": "${var.rds_db_name}"
-      },
-      {
-        "name": "RDS_HOSTNAME",
-        "value": "${var.rds_hostname}"
-      },
-      {
-        "name": "RDS_USERNAME",
-        "value": "${var.rds_username}"
-      },
-      {
-        "name": "RDS_PASSWORD",
-        "value": "${var.rds_password}"
-      },
-      {
-        "name": "DYNAMODB_ENDPOINT",
-        "value": "dynamodb.us-east-1.amazonaws.com"
-      },
-      {
-        "name": "SNS_ENDPOINT",
-        "value": "sns.us-east-1.amazonaws.com"
-      },
-      {
-        "name": "AWS_ACCOUNT_ID",
-        "value": "${data.aws_caller_identity.current.account_id}"
-      },
-      {
-        "name": "AWS_REGION",
-        "value": "${data.aws_region.current.name}"
-      },
-      {
-        "name": "AWS_ACCESS_KEY",
-        "value": "${var.aws_access_key}"
-      },
-      {
-        "name": "AWS_SECRET_KEY",
-        "value": "${var.aws_secret_key}"
-      },
-      {
         "name": "ENVIRONMENT",
         "value": "${var.environment}"
       },
       {
-        "name": "PORT",
-        "value": "${var.port}"
-      },
-      {
-        "name": "GITHUB_OAUTH_TOKEN",
-        "value": "${var.oauth_token}"
+        "name": "AWS_REGION",
+        "value": "${data.aws_region.current.name}"
       }
     ]
   }
 ]
 DEFINITION
 }
 
+locals {
+  default_parameters = {
+    PORT = "${var.port}"
+    ENVIRONMENT = "${var.environment}"
+    RDS_DB_NAME = "${var.rds_db_name}"
+    RDS_HOSTNAME = "${var.rds_hostname}"
+    RDS_USERNAME = "${var.rds_username}"
+    DYNAMODB_ENDPOINT = "dynamodb.us-east-1.amazonaws.com"
+    SNS_ENDPOINT = "sns.us-east-1.amazonaws.com"
+    AWS_ACCOUNT_ID = "${data.aws_caller_identity.current.account_id}"
+    AWS_ACCESS_KEY = "${var.aws_access_key}"
+    HUD_AI_BASE_API_URL = "https://${var.environment}.api.hud.ai/v1"
```

Comment:
let's remove this, the idea here is we can use the infrastructure repo for other projects (keep everything HUD agnostic)

Reply:
sounds good